internal/cli/up.go

```diff
@@ -245,20 +245,21 @@ func provideWebURL(webHost model.WebHost, webPort model.WebPort) (model.WebURL,
 }
 
 func provideAssetServer(mode model.WebMode, version model.WebVersion) (assets.Server, error) {
+	s, err := assets.NewEmbeddedServer()
```

Comment:
this error is getting dropped on the floor

Reply:
I think based on your comment below that this would work better as `s, ok := ...` instead